containers/podman-desktop
# feat: add confirmation dialog when deleting objects

feat: add confirmation dialog when deleting objects

### What does this PR do?

Adds confirmation dialogs to all critical components of Podman Desktop
where you are deleting a container, pod, deployment, service, etc.

This makes sure you do not accidently delete something (such as a
deployment) that could be critical infrastructure.

### Screenshot / video of UI




https://github.com/containers/podman-desktop/assets/6422176/b094db26-d9d6-4d8d-8891-16a05c39b4a8



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop/issues/5444

### How to test this PR?

Try to delete anything (volume, pod, container, etc.) and you will now
see a dialog.



Signed-off-by: Charlie Drage <charlie@charliedrage.com>
